projects/cds/src/profiletag/services/profile-tag.injector.service.ts

```diff
@@ -17,12 +19,15 @@ export class ProfileTagInjectorService {
   private injectionsEvents$: Observable<boolean> = merge(
     this.notifyProfileTagOfConsentGranted(),
     this.notifyProfileTagOfCartChange(),
-    this.notifyProfileTagOfPageLoaded()
+    this.notifyProfileTagOfPageLoaded(),
+    this.notifyEcOfLoginSuccessful()
   );
 
   constructor(
     private profileTagEventTracker: ProfileTagEventService,
-    private spartacusEventTracker: SpartacusEventService
+    private spartacusEventTracker: SpartacusEventService,
+    protected http: HttpClient,
```

Comment:
You should follow our connector pattern: https://sap.github.io/cloud-commerce-spartacus-storefront-docs/connecting-to-other-systems/#connector. This pattern allows extensibility.
Basically, you should create new connector and adapter classes. The connector should inject the adapter, and the adapter should issue http calls. For an example you can check out `projects/cds/src/merchandising/facade/cds-merchandising-product.service.ts`.

Reply:
Implemented. Please let me know if it still doesn't look right!

Reply:
Thanks! Looks good.